### PR TITLE
Document TUI command construction using Python journaling

### PR DESCRIPTION
**Pull request by mkundu1**
screencapture-file-D-ANSYSDev-PyFluentDev-pyfluent-doc-build-html-user-guide-tui-commands-html-2023-09-26-16_05_29

### CODE REVIEW

**Review by prmukherj — APPROVED** [review]

File: doc/source/user_guide/tui_commands.rst [hunk 1]
```diff
@@ -22,79 +22,73 @@ you can use the Python built-in `help <https://docs.python.org/3/library/functio
...
 
 The arguments to a TUI command are those that would be passed in direct
-interaction in the Fluent console, but they are in a Pythonic style. The most
-productive way to write Python commands is with reference to existing TUI
-commands. The following examples show how the Python usage mirrors the existing
-TUI usage.
+interaction in the Fluent console, but they are in a Pythonic style. In the recent
+Fluent versions, in both meshing and solution mode,
+you can use Python journaling, which is a beta feature,
+to construct the TUI commands for PyFluent. The following section describes how to
```
Comment (raph-luc) [inline doc/source/user_guide/tui_commands.rst:28]: Should we mention somewhere how to perform Python journaling?

As far as I've seen, we don't have that info anywhere in the PyFluent documentation. The current PyFluent journaling documentation seems to be mostly useless/empty: 
https://fluent.docs.pyansys.com/version/stable/api/general/index.html#journaling

**Review by hpohekar — APPROVED** [review]

File: doc/source/user_guide/tui_commands.rst [hunk 2]
```diff
@@ -22,79 +22,82 @@ you can use the Python built-in `help <https://docs.python.org/3/library/functio
...
+construct the TUI commands for PyFluent in different Fluent versions.
 
 TUI command construction
--------------------------
-Assume that you are in the solution mode and type the following in the Fluent
-console to set velocity inlet properties:
+------------------------
+
+In Fluent 2023 R2, the recorded Python journal from Fluent contains the Python
```
Comment (seanpearsonuk) [inline doc/source/user_guide/tui_commands.rst:34]:
```suggestion
From Fluent 2023 R2, the recorded Python journal from Fluent contains the Python
```

File: doc/source/user_guide/tui_commands.rst [hunk 3]
```diff
@@ -22,79 +22,82 @@ you can use the Python built-in `help <https://docs.python.org/3/library/functio
...
+commands corresponding to the TUI commands executed in Fluent. The TUI commands for
+which settings API exist are converted to the corresponding settings API commands.
```
Comment (seanpearsonuk) [inline doc/source/user_guide/tui_commands.rst:36]:
```suggestion
commands corresponding to the TUI commands executed in Fluent. The TUI command conversion will
generate a call to a corresponding settings API command if one exists.
```

File: doc/source/user_guide/tui_commands.rst [hunk 4]
```diff
@@ -22,79 +22,82 @@ you can use the Python built-in `help <https://docs.python.org/3/library/functio
...
+If you start the Python journaling in the solution mode and type the following
+in the Fluent console to set velocity inlet properties:
```
Comment (seanpearsonuk) [inline doc/source/user_guide/tui_commands.rst:38]:
```suggestion
From the 2023 R2 release onward, a Fluent Python journal contains Python calls corresponding to the TUI commands executed in Fluent. Python journaling will
generates a call to a corresponding settings API command if one exists.
For instance, with Python started in solution mode, you can type following in the Fluent console to set velocity inlet properties:
```

File: doc/source/user_guide/tui_commands.rst (hunk 1 above)
Comment (mkundu1) [reply doc/source/user_guide/tui_commands.rst:28]: This is part of Fluent documentation.

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**